### PR TITLE
Add security descriptor file samples for System Automation

### PR DESCRIPTION
**Pull request by juergen-holtz**

### CODE REVIEW

**Comment by juergen-holtz** [comment]
Hello Hiren @shreeji818 can you review my additions so that the merge into the `zOS-Automation` directory of the repository can continue? This reflects all our discussions in the past weeks.

File: zOS-Automation/zOSMF-SCA/ing_admin.json [hunk 1]
```diff
@@ -0,0 +1,2418 @@
...
+      "ResourceClass": "SYSAUTO",
+      "WhoNeedsAccess": "Administrators",
+      "LevelOfAccessRequired": "UPDATE",
+      "ItemDescription": "Controls the release function of the INGPAC command"
+    },
+    {
+      "ItemID": "5698SA40102I0015000",
+      "ItemType": "SEMI-PROGRAMMABLE",
+      "ItemCategory": "product-wide, general usage",
+      "ResourceProfile": "POLICY.<enterprise-name>.OPTIONS.ACTIVITYLOG",
```
Comment (shreeji818) [inline zOS-Automation/zOSMF-SCA/ing_admin.json:81]: Should we add enterprise-name in cmds.txt?

**Review by shreeji818 — APPROVED** [review]
I added one comment. I am approving the PR. So, no need to resubmit.

**Review by shreeji818 — APPROVED** [review]

**Comment by juergen-holtz** [comment]
Hello @shreeji818 , I've added the `<enterprise-name>` variable in `cmds.txt` so that the customer can also globally replace this one if he likes to.  This is in commit 680c969.

Can you, please approve and merge the PR?  I am not authorized for this.

**Review by shreeji818 — APPROVED** [review]
Looks Good. Thank you

**Pull request merged**